experiments/prediction/figure1.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+
+from RlGlue import RlGlue
+from utils.Collector import Collector
+from utils.policies import actionArrayToPolicy
+from utils.rl_glue import RlGlueCompatWrapper
```

Comment:
Again, I understand that these are useful but if we could avoid using anything that the reader cannot see the code for, that might have been better? I'm not sure though.

Reply:
They can see the code for this :)